packages/react/test/specs/lib/felaExpandCssShorthandsPlugin-test.ts

```diff
@@ -0,0 +1,67 @@
+import felaExpandCssShorthandsPlugin from 'src/lib/felaExpandCssShorthandsPlugin'
+
+const expandCssShorthands = felaExpandCssShorthandsPlugin()
+
+describe('felaExpandCssShorthandsPlugin', () => {
```

Comment:
nice tests 👍 couple of things:

1. I'd use `toMatchObject` instead of `toEqual` to compare objects so that we don't have to rely on key order as that is a detail of the CSS library and might change; we're also not interested in the order

2. I'd also add a test for what happens when we use an expanded prop together with its shorthand; something like:

```typescript
test('should merge expanded prop with its shorthand', () => {
  const style = {
    marginTop: '3px',
    margin: '10px',
    marginRight: '15px',
  }

  expect(expandCssShorthands(style)).toMatchObject({
    marginTop: '10px', // overridden by margin: '10px'
    marginRight: '15px', // overridden by marginRight: '15px'
    marginBottom: '10px',
    marginLeft: '10px',
  })
})
```

Reply:
Added, actually this is really good case for corner cases, and testing whether one style object within itself will correctly apply the styles. Thanks for the suggestion on this one @Bugaa92